packages/vitest/src/integrations/chai/jest-expect.ts

```diff
@@ -160,11 +160,61 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
       obj,
     )
   })
-  def('toBeGreaterThan', function(expected: number) {
-    return this.to.greaterThan(expected)
+  def('toBeGreaterThan', function(expected: number | bigint) {
+    const actual = this._obj
+    if (typeof actual !== 'number' && typeof actual !== 'bigint') {
+      return this.assert(
```

Comment:
Not sure about this use of `assert`

It's not actually an `AssertionError`, since no assertion was actually performed bc the input was wrong. Maybe we can 
1. Throw TypeError
2. Move it to some helper function to reuse in other assertions